tools/flang1/flang1exe/main.c

```diff
@@ -127,7 +127,11 @@ static char *accel = NULL;
 
 /** Product name in debug output
  */
+#ifdef FLANG_VENDOR
+#define DNAME FLANG_VENDOR "F90"
```

Comment:
Instead of hardcoding "F90" here, can we use `version.lang` instead?

Reply:
`"F90"` is used here only because `DNAME` is otherwise defined as `"F90"` (see few lines below)

Reply:
Yeah, I understand that. What I meant was that we should just replace `DNAME` with `version.lang`.

Reply:
The way DNAME is used rules out the use of version.lang here. What I could do was to make sure that both version.lang and DNAME are referring the same name.